tests/data/dags/simple_test_dag

```diff
@@ -49,6 +49,7 @@ with DAG(
     doc_md=docs,
     tags=tags,
 ) as dag:
+
```

Comment:
Required due to [failures in the CI](https://app.circleci.com/pipelines/github/mozilla/bigquery-etl/30034/workflows/57ae574d-6adb-4afa-aaac-ba92d322a6e6/jobs/324284)